src/client/debug/debug.js

```diff
@@ -16,6 +16,7 @@ import { PlayerInfo } from './playerinfo';
 import { DebugMove } from './debug-move';
 import { GameLog } from '../log/log';
 import { restore } from '../../core/action-creators';
+import { parse, stringify } from 'flatted/cjs';
```

Comment:
I think it should be `flatted/esm`

From https://github.com/WebReflection/flatted#flatted
```javascript
// ESM
import {parse, stringify} from 'flatted/esm';

// CJS
const {parse, stringify} = require('flatted/cjs');

```

Reply:
I aggree. I changed to esm locally, and changed the rollup config accordingly. However, now tests do not run anymore because they fail to resolve the "export" keyword inside flatted/esm/index.js.

Somehow jest now needs to be told to transform ES6 modules also. Likely another change in .babelrc is necessary.

Reply:
So how was I able to push this branch at all - it should have been caught by the prepush hook? Since I can't push the esm variant because now tests are failing :(

As far as I understand this, it should work since .babelrc contains `transform-es2015-modules-commonjs` inside test env directly.

Reply:
Sometimes tests can pass locally but fail on Travis if it depends on something in node_modules that's set up correctly locally manually.

Reply:
Oh I thought your question was about how to push now that the tests are failing.